maintainers/README.md

```diff
@@ -172,3 +172,39 @@ to the team without an approval by at least one existing member.
 Various utility scripts, which are mainly useful for nixpkgs maintainers,
 are available under `./scripts/`.  See its [README](./scripts/README.md)
 for further information.
+
+# Guidelines for Committers
+
+When merging changes, care must be taken to reduce impact to the `master`
+branch. If a commit breaks evaluation in `master` it could make it into new PR
+branches and cause further breakage.
```

Comment:
```suggestion
branch. If a commit breaks evaluation, it will affect Ofborg evaluation results
in other pull requests and block hydra CI, thus introducing chaos to our workflow.
```